src/diffusers/loaders.py

```diff
@@ -1005,8 +1061,12 @@ def load_lora_into_unet(cls, state_dict, network_alpha, unet):
             warn_message = "You have saved the LoRA weights using the old format. To convert the old LoRA weights to the new format, you can first load them in a dictionary and then create a new dictionary like the following: `new_state_dict = {f'unet'.{module_name}: params for module_name, params in old_state_dict.items()}`."
             warnings.warn(warn_message)
 
+        if state_dict_aux:
+            unet._load_lora_aux(state_dict_aux, network_alpha=network_alpha)
+            unet.aux_state_dict_populated = True
+
```

Comment:
+1 to what patrick said, is it possible to just have one state dict instead of an auxiliary one?